### ISSUE TITLE
Small legacy icons

### ISSUE THREAD

**Issue by giovannicaligaris** [eth-p/wingpanel-indicator-ayatana#9]
I just finished building the latest master and the bluetooth icon works just fine. Good job!.

I have noticed that the legacy icons have become smaller now, just a tiny bit. 

![screenshot from 2018-10-19 14 33 53](https://user-images.githubusercontent.com/26255347/47221358-07976c80-d3ac-11e8-94e2-f5127b34af21.png)

**Comment by giovannicaligaris** [comment]
Fixed it.

const int IDEAL_ICON_SIZE = 18;

has to be changed to 24 in src/AyatanaIndicator.vala

**Comment by eth-p** [comment]
Yeah... I need to find a way to properly handle icon sizes. In my installation, indicators from some apps (e.g. Google Play Music Desktop Player or TeamViewer) consume the entire vertical space of wingpanel, while others don't.

I was thinking of analyzing the icons when they're added and adjusting the ideal size based on the empty space in the icon file/Pixbuf.
